nab/scorer.py

```diff
@@ -102,12 +102,12 @@ def scoreCorpus(threshold, args):
 
   # Using `map_async` instead of `map` so interrupts are properly handled.
   # See: http://stackoverflow.com/a/1408476
-  results = pool.map_async(scoreDataSet, args).get(99999999)
+  results = pool.map_async(scoreDataSet, args).get(999999)
 
   # Total the 6 scoring metrics for all data files
   totals = [None]*3 + [0]*6
   for row in results:
-    for i in xrange(6):
+    for i in range(6):
```

Comment:
magic "6", NUMBER_OF_XXX?

Reply:
The comment on line 107 explains somewhat do you feel that is insufficient?

Reply:
>  line 107 explains somewhat do you feel that is insufficient? 

that should be enough, I must have missed it while reviewing. Thank you